src/server/game/Battlegrounds/Zones/BattlegroundDS.cpp

```diff
@@ -173,10 +174,10 @@ bool BattlegroundDS::HandlePlayerUnderMap(Player* player)
     return true;
 }
 
-void BattlegroundDS::FillInitialWorldStates(WorldPacket& data)
+void BattlegroundDS::FillInitialWorldStates(WorldPackets::WorldState::InitWorldStates& packet)
 {
-    data << uint32(3610) << uint32(1);                                              // 9 show
-    Arena::FillInitialWorldStates(data);
+    packet.Worldstates.emplace_back(0xe1a, 1); // ARENA_WORLD_STATE_ALIVE_PLAYERS_SHOW
```

Comment:
Well here it was decimal and you made it into hex now :p not sure if there's a prereference

Reply:
I replaced all instances with hex format since hex is commonly used to represent packet data. It’s probably best to use hex for consistency